inc/switchtec/mfg.h

```diff
@@ -92,6 +92,30 @@ struct switchtec_security_cfg_state {
 	uint8_t public_key[SWITCHTEC_KMSK_NUM][SWITCHTEC_KMSK_LEN];
 };
 
+/**
+ * @brief Flag which indicates if an OTP region is programmable or not
+ */
+enum switchtec_otp_program_status {
+	SWITCHTEC_OTP_PROGRAMMABLE = 0,
+	SWITCHTEC_OTP_UNPROGRAMMABLE = 1,
+};
+
+struct switchtec_security_cfg_otp_region {
+	bool basic_valid, mixed_ver_valid, main_fw_ver_valid,
+	     sec_unlock_ver_valid, kmsk_valid[4];
+	enum switchtec_otp_program_status basic, mixed_ver, main_fw_ver,
+					  sec_unlock_ver, kmsk[4];
```

Comment:
Nit: Prefer one line per member in a struct.

Reply:
Updated.